gwsurrogate/surrogate.py

```diff
@@ -2449,6 +2506,216 @@ def _get_intrinsic_parameters(self, q, chiA0, chiB0, precessing_opts,
+                            0.36231884057971003]
+
+            q_fit_offset    = q_fit_offsets[subdomain_ID]
+            q_fit_slope     = q_fit_slopes[subdomain_ID]
+            q_max_bfOrder   = 3
+            chi_max_bfOrder = 2
+            return q_fit_offset, q_fit_slope, q_max_bfOrder, chi_max_bfOrder
+
+        def get_subdomain_ID(q, chiA0, chiB0):
+            """ Get the surrogate subdomain ID for evaluation point in
+             parameter space. 
+             
+            INPUT
+            =====
+            q :         Mass ratio, mA/mB >= 1.
+            chiA0:      Dimensionless spin vector of the heavier black hole.
+            chiB0:      Dimensionless spin vector of the lighter black hole.
+
+            IMPORTANT NOTES:
+            ================
+
+            Subdomains are defined based on the effective spin parameter 
+            as computed in the early inspiral. While the effective spin
+            parameter is nearly constant throughout the inspiral it can change 
+            by small amounts. Surrogates built in each subdomain are valid over
+            a larger region of the parameter space than the subdomain itself,
+            which helps in ensuring the model is accurate when evaluated
+            on a subdomain boundary and with spins specified at different
+            times/frequencies throughout the inspiral.
+            """
+            chi1z = chiA0[2]
+            chi2z = chiB0[2]
+            chi_eff = (q*chi1z + chi2z) / (q + 1.)
+            # print("q, chi_eff: ", q, chi_eff)
+            if q <= 2.:
+                return 0
+            if q <= 4.:
+                return 1
+            if q <= 7. and chi_eff <= -0.3:
+                return 2
+            if q <= 7. and chi_eff <= 0.3:
+                return 3
+            if q <= 7. and chi_eff > 0.3:
+                return 4
+            if q <= 10. and chi_eff <= -0.3:
+                return 5
+            if q <= 10. and chi_eff <= 0.3:
+                return 6
+            if q <= 10. and chi_eff > 0.3:
+                return 7
+            if q <= 15. and chi_eff <= -0.3:
+                return 8
+            if q <= 15. and chi_eff <= 0.3:
+                return 9
+            if q <= 15. and chi_eff > 0.3:
+                return 10
+            if q > 15. and chi_eff <= -0.3:
+                return 11
+            if q > 15. and chi_eff <= 0.3:
+                return 12
+            if q > 15. and chi_eff > 0.3:
+                return 13
+            
+            raise ValueError("Domain not found for q={}, chi1z={}, chi2z={}".format(q, chi1z, chi2z))
+
+
+        # largest ell mode in the surrogate
+        ellMax_SEOBNRv4PHMSur = 5
+
+        # max allowable reference dimensionless orbital angular frequency
+        omega_ref_max_SEOBNRv4PHMSur = 0.21
+
+        # number of subdomains (should be maximum possible return of get_subdomain_ID + 1)
+        num_subdomains_SEOBNRv4PHMSur = 14
```

Comment:
Is there a good argument for naming these variables with `_SEOBNRv4PHMSur` at the end, and similarly `ellMax_NRSur7dq4, omega_ref_max_NRSur7dq4` in that class?